diff -up php-7.3.0RC1/sapi/apache2handler/php_functions.c.omv~ php-7.3.0RC1/sapi/apache2handler/php_functions.c
--- php-7.3.0RC1/sapi/apache2handler/php_functions.c.omv~	2018-09-13 23:54:45.499115056 +0200
+++ php-7.3.0RC1/sapi/apache2handler/php_functions.c	2018-09-13 23:55:15.453241974 +0200
@@ -225,10 +225,7 @@ PHP_FUNCTION(apache_get_output_filters)
                WRONG_PARAM_COUNT;
        }
 
-       if(array_init(return_value) != SUCCESS)
-       {
-               RETURN_NULL();
-       }
+       array_init(return_value);
 
        ctx = SG(server_context);
 
