diff -up php-7.3.0RC1/ext/imap/config.m4.no_pam_in_c-client.droplet php-7.3.0RC1/ext/imap/config.m4
--- php-7.3.0RC1/ext/imap/config.m4.no_pam_in_c-client.droplet	2018-09-13 23:20:57.043938103 +0200
+++ php-7.3.0RC1/ext/imap/config.m4	2018-09-13 23:21:32.613041390 +0200
@@ -178,12 +178,6 @@ if test "$PHP_IMAP" != "no"; then
     ],[])
     CPPFLAGS=$old_CPPFLAGS
 
-    PHP_CHECK_LIBRARY(pam, pam_start,
-    [
-      PHP_ADD_LIBRARY(pam,, IMAP_SHARED_LIBADD)
-      AC_DEFINE(HAVE_LIBPAM,1,[ ])
-    ])
-
     PHP_CHECK_LIBRARY(crypt, crypt,
     [
       PHP_ADD_LIBRARY(crypt,, IMAP_SHARED_LIBADD)
