diff -up php-8.4.1/Zend/zend_cpuinfo.h.3~ php-8.4.1/Zend/zend_cpuinfo.h
--- php-8.4.1/Zend/zend_cpuinfo.h.3~	2024-11-21 19:45:58.787896144 +0100
+++ php-8.4.1/Zend/zend_cpuinfo.h	2024-11-21 19:55:11.302381325 +0100
@@ -266,7 +266,16 @@ static inline int zend_cpu_supports_clde
 #ifdef PHP_HAVE_BUILTIN_CPU_INIT
 	__builtin_cpu_init();
 #endif
+#if defined(__clang__) && defined(__x86_64__)
+    // bit 25 of the ECX register in sub-leaf 07H of CPUID
+    uint32_t op = 7, eax, ebx, ecx, edx;
+    asm volatile("cpuid\n"
+               : "=a" (eax), "=b" (ebx), "=c" (ecx), "=d" (edx)
+               : "a" (op));
+    return ecx & 0x2000000;
+#else
 	return __builtin_cpu_supports("cldemote");
+#endif
 }
 #endif
 
diff -up php-8.4.1/ext/opcache/jit/zend_jit.c.3~ php-8.4.1/ext/opcache/jit/zend_jit.c
