diff -up php-src-php-8.1.1RC1/ext/pdo_mysql/mysql_driver.c.omv~ php-src-php-8.1.1RC1/ext/pdo_mysql/mysql_driver.c
--- php-src-php-8.1.1RC1/ext/pdo_mysql/mysql_driver.c.omv~	2021-12-05 21:00:58.980985356 +0100
+++ php-src-php-8.1.1RC1/ext/pdo_mysql/mysql_driver.c	2021-12-05 21:01:57.277153122 +0100
@@ -544,7 +544,7 @@ static int pdo_mysql_get_attribute(pdo_d
 			ZVAL_BOOL(return_value, H->local_infile);
 			break;
 
-#if MYSQL_VERSION_ID >= 80021 || defined(PDO_USE_MYSQLND)
+#if (MYSQL_VERSION_ID >= 80021 && !defined(MARIADB_BASE_VERSION)) || defined(PDO_USE_MYSQLND)
 		case PDO_MYSQL_ATTR_LOCAL_INFILE_DIRECTORY:
 		{
 			const char* local_infile_directory = NULL;
@@ -765,7 +765,7 @@ static int pdo_mysql_handle_factory(pdo_
 #endif
 		}
 
-#if MYSQL_VERSION_ID >= 80021 || defined(PDO_USE_MYSQLND)
+#if (MYSQL_VERSION_ID >= 80021 && !defined(MARIADB_BASE_VERSION)) || defined(PDO_USE_MYSQLND)
 		zend_string *local_infile_directory = pdo_attr_strval(driver_options, PDO_MYSQL_ATTR_LOCAL_INFILE_DIRECTORY, NULL);
 		if (local_infile_directory && !php_check_open_basedir(ZSTR_VAL(local_infile_directory))) {
 			if (mysql_options(H->server, MYSQL_OPT_LOAD_DATA_LOCAL_DIR, (const char *)ZSTR_VAL(local_infile_directory))) {
