diff -up php-7.3.1/ext/date/php_date.c.0014~ php-7.3.1/ext/date/php_date.c
--- php-7.3.1/ext/date/php_date.c.0014~	2019-01-08 14:56:00.000000000 +0100
+++ php-7.3.1/ext/date/php_date.c	2019-01-10 20:03:08.531475574 +0100
@@ -684,7 +684,7 @@ static zval *date_period_read_property(z
 static void date_period_write_property(zval *object, zval *member, zval *value, void **cache_slot);
 
 /* {{{ Module struct */
-zend_module_entry date_module_entry = {
+__attribute__((visibility("default"))) zend_module_entry date_module_entry = {
 	STANDARD_MODULE_HEADER_EX,
 	NULL,
 	NULL,
diff -up php-7.3.1/ext/libxml/libxml.c.0014~ php-7.3.1/ext/libxml/libxml.c
--- php-7.3.1/ext/libxml/libxml.c.0014~	2019-01-08 14:56:00.000000000 +0100
+++ php-7.3.1/ext/libxml/libxml.c	2019-01-10 20:03:08.531475574 +0100
@@ -133,7 +133,7 @@ static const zend_function_entry libxml_
 	PHP_FE_END
 };
 
-zend_module_entry libxml_module_entry = {
+__attribute__((visibility("default"))) zend_module_entry libxml_module_entry = {
 	STANDARD_MODULE_HEADER,
 	"libxml",                /* extension name */
 	libxml_functions,        /* extension function list */
diff -up php-7.3.1/ext/pcre/php_pcre.c.0014~ php-7.3.1/ext/pcre/php_pcre.c
--- php-7.3.1/ext/pcre/php_pcre.c.0014~	2019-01-08 14:55:58.000000000 +0100
+++ php-7.3.1/ext/pcre/php_pcre.c	2019-01-10 20:03:08.532475585 +0100
@@ -3000,7 +3000,7 @@ static const zend_function_entry pcre_fu
 	PHP_FE_END
 };
 
-zend_module_entry pcre_module_entry = {
+__attribute__((visibility("default"))) zend_module_entry pcre_module_entry = {
 	STANDARD_MODULE_HEADER,
    "pcre",
 	pcre_functions,
diff -up php-7.3.1/ext/reflection/php_reflection.c.0014~ php-7.3.1/ext/reflection/php_reflection.c
--- php-7.3.1/ext/reflection/php_reflection.c.0014~	2019-01-08 14:55:54.000000000 +0100
+++ php-7.3.1/ext/reflection/php_reflection.c	2019-01-10 20:03:08.532475585 +0100
@@ -6719,7 +6719,7 @@ PHP_MINFO_FUNCTION(reflection) /* {{{ */
 	php_info_print_table_end();
 } /* }}} */
 
-zend_module_entry reflection_module_entry = { /* {{{ */
+__attribute__((visibility("default"))) zend_module_entry reflection_module_entry = { /* {{{ */
 	STANDARD_MODULE_HEADER,
 	"Reflection",
 	reflection_ext_functions,
diff -up php-7.3.1/ext/simplexml/simplexml.c.0014~ php-7.3.1/ext/simplexml/simplexml.c
--- php-7.3.1/ext/simplexml/simplexml.c.0014~	2019-01-08 14:55:58.000000000 +0100
+++ php-7.3.1/ext/simplexml/simplexml.c	2019-01-10 20:03:08.532475585 +0100
@@ -2634,7 +2634,7 @@ static const zend_module_dep simplexml_d
 };
 /* }}} */
 
-zend_module_entry simplexml_module_entry = { /* {{{ */
+__attribute__((visibility("default"))) zend_module_entry simplexml_module_entry = { /* {{{ */
 	STANDARD_MODULE_HEADER_EX, NULL,
 	simplexml_deps,
 	"SimpleXML",
diff -up php-7.3.1/ext/spl/php_spl.c.0014~ php-7.3.1/ext/spl/php_spl.c
--- php-7.3.1/ext/spl/php_spl.c.0014~	2019-01-08 14:55:55.000000000 +0100
+++ php-7.3.1/ext/spl/php_spl.c	2019-01-10 20:03:08.533475595 +0100
@@ -1037,7 +1037,7 @@ PHP_RSHUTDOWN_FUNCTION(spl) /* {{{ */
 
 /* {{{ spl_module_entry
  */
-zend_module_entry spl_module_entry = {
+__attribute__((visibility("default"))) zend_module_entry spl_module_entry = {
 	STANDARD_MODULE_HEADER,
 	"SPL",
 	spl_functions,
diff -up php-7.3.1/ext/standard/basic_functions.c.0014~ php-7.3.1/ext/standard/basic_functions.c
--- php-7.3.1/ext/standard/basic_functions.c.0014~	2019-01-08 14:55:58.000000000 +0100
+++ php-7.3.1/ext/standard/basic_functions.c	2019-01-10 20:03:08.533475595 +0100
@@ -3451,7 +3451,7 @@ static const zend_module_dep standard_de
 };
 /* }}} */
 
-zend_module_entry basic_functions_module = { /* {{{ */
+__attribute__((visibility("default"))) zend_module_entry basic_functions_module = { /* {{{ */
 	STANDARD_MODULE_HEADER_EX,
 	NULL,
 	standard_deps,
diff -up php-7.3.1/Zend/zend_signal.c.0014~ php-7.3.1/Zend/zend_signal.c
--- php-7.3.1/Zend/zend_signal.c.0014~	2019-01-08 14:56:01.000000000 +0100
+++ php-7.3.1/Zend/zend_signal.c	2019-01-10 20:07:05.070135949 +0100
@@ -375,7 +375,7 @@ static void zend_signal_globals_ctor(zen
 }
 /* }}} */
 
-void zend_signal_init(void) /* {{{ */
+__attribute__((visibility("default"))) void zend_signal_init(void) /* {{{ */
 {
 	int signo;
 	struct sigaction sa;
@@ -397,7 +397,7 @@ void zend_signal_init(void) /* {{{ */
 
 /* {{{ zend_signal_startup
  * alloc zend signal globals */
-ZEND_API void zend_signal_startup(void)
+__attribute__((visibility("default"))) ZEND_API void zend_signal_startup(void)
 {
 
 #ifdef ZTS
