diff -up php-7.3.0RC1/build/build2.mk.phpize.droplet php-7.3.0RC1/build/build2.mk
--- php-7.3.0RC1/build/build2.mk.phpize.droplet	2018-09-13 22:42:02.479901557 +0200
+++ php-7.3.0RC1/build/build2.mk	2018-09-13 22:46:57.574353336 +0200
@@ -26,6 +26,7 @@ targets = $(TOUCH_FILES) configure $(con
 
 PHP_AUTOCONF ?= 'autoconf'
 PHP_AUTOHEADER ?= 'autoheader'
+PHP_ACLOCAL ?= 'aclocal'
 
 SUPPRESS_WARNINGS ?= 2>&1 | (egrep -v '(AC_PROG_CXXCPP was called before AC_PROG_CXX|defined in acinclude.m4 but never used)'||true)
 
diff -up php-7.3.0RC1/scripts/phpize.in.phpize.droplet php-7.3.0RC1/scripts/phpize.in
--- php-7.3.0RC1/scripts/phpize.in.phpize.droplet	2018-09-11 13:13:29.000000000 +0200
+++ php-7.3.0RC1/scripts/phpize.in	2018-09-13 22:47:29.701503740 +0200
@@ -11,7 +11,7 @@ SED="@SED@"
 
 FILES_BUILD="mkdep.awk scan_makefile_in.awk shtool libtool.m4 ax_check_compile_flag.m4 ax_gcc_func_attribute.m4"
 FILES="acinclude.m4 Makefile.global config.sub config.guess ltmain.sh run-tests*.php"
-CLEAN_FILES="$FILES *.o *.lo *.la .libs/ build/ modules/ install-sh \
+CLEAN_FILES="$FILES *.o *.so *.lo *.la .libs/ build/ modules/ install-sh \
 	mkinstalldirs missing config.nice config.sub config.guess configure configure.ac \
 	aclocal.m4 config.h config.h.in conftest* ltmain.sh libtool config.cache autom4te.cache/ \
 	config.log config.status Makefile Makefile.fragments Makefile.objects confdefs.h \
@@ -121,6 +121,7 @@ phpize_check_autotools()
 {
   test -z "$PHP_AUTOCONF" && PHP_AUTOCONF=autoconf
   test -z "$PHP_AUTOHEADER" && PHP_AUTOHEADER=autoheader
+  test -z "$PHP_ACLOCAL" && PHP_ACLOCAL=aclocal
 
   if test ! -x "$PHP_AUTOCONF" && test ! -x "`$php_shtool path $PHP_AUTOCONF`"; then
     cat <<EOF
@@ -138,6 +139,14 @@ Cannot find autoheader. Please check you
 EOF
     exit 1
   fi
+  if ! test -x "`$php_shtool path $PHP_ACLOCAL`"; then
+    cat <<EOF
+Cannot find aclocal. Please check your autoconf installation and the \$PHP_ACLOCAL 
+environment variable is set correctly and then rerun this script. 
+
+EOF
+    exit 1
+  fi
 }
 
 phpize_copy_files()
@@ -151,13 +160,13 @@ phpize_copy_files()
 
 phpize_replace_prefix()
 {
-  $SED \
-  -e "s#@prefix@#$prefix#" \
-  < "$phpdir/phpize.m4" > configure.ac
+  cp $phpdir/phpize.m4 configure.in
 }
 
 phpize_autotools()
 {
+  libtoolize --copy --force || exit 1
+  $PHP_ACLOCAL    || exit 1
   $PHP_AUTOCONF   || exit 1
   $PHP_AUTOHEADER || exit 1
 }
