diff -up php-8.4.1/ext/dba/config.m4.1~ php-8.4.1/ext/dba/config.m4
--- php-8.4.1/ext/dba/config.m4.1~	2024-11-21 19:36:37.963196077 +0100
+++ php-8.4.1/ext/dba/config.m4	2024-11-21 19:38:52.524347644 +0100
@@ -339,7 +339,7 @@ AC_DEFUN([PHP_DBA_DB_CHECK],[
           [$3;])],
           [AC_EGREP_CPP([yes], [
 #include "$THIS_INCLUDE"
-#if DB_VERSION_MAJOR == $1 || ($1 == 4 && DB_VERSION_MAJOR == 5)
+#if DB_VERSION_MAJOR == $1 || ($1 == 4 && DB_VERSION_MAJOR >= 5)
             yes
 #endif
           ],[
@@ -477,6 +477,10 @@ if test "$PHP_DB4" != "no"; then
   done
   PHP_DBA_DB_CHECK([4],
     [
+      db-18.1
+      db-6.2
+      db-6.1
+      db-6.0
       db-5.3
       db-5.1
       db-5.0
