diff -up php-7.4.11/ext/mysqli/mysqli_api.c.mariadb~ php-7.4.11/ext/mysqli/mysqli_api.c
--- php-7.4.11/ext/mysqli/mysqli_api.c.mariadb~	2020-09-29 12:17:09.000000000 +0200
+++ php-7.4.11/ext/mysqli/mysqli_api.c	2020-10-06 12:49:49.534565626 +0200
@@ -614,7 +614,7 @@ PHP_FUNCTION(mysqli_change_user)
 	size_t			user_len, password_len, dbname_len;
 	zend_ulong		rc;
 #if !defined(MYSQLI_USE_MYSQLND) && defined(HAVE_MYSQLI_SET_CHARSET)
-	const		CHARSET_INFO * old_charset;
+	const		MY_CHARSET_INFO * old_charset;
 #endif
 
 	if (zend_parse_method_parameters(ZEND_NUM_ARGS(), getThis(), "Osss!", &mysql_link, mysqli_link_class_entry, &user, &user_len, &password, &password_len, &dbname, &dbname_len) == FAILURE) {
diff -up php-7.4.11/ext/mysqli/mysqli.c.mariadb~ php-7.4.11/ext/mysqli/mysqli.c
--- php-7.4.11/ext/mysqli/mysqli.c.mariadb~	2020-09-29 12:17:09.000000000 +0200
+++ php-7.4.11/ext/mysqli/mysqli.c	2020-10-06 12:49:49.534565626 +0200
@@ -24,6 +24,7 @@
 
 #include <signal.h>
 
+#include <server/mysql_version.h>
 #include "php.h"
 #include "php_ini.h"
 #include "ext/standard/info.h"
diff -up php-7.4.11/ext/mysqli/mysqli_nonapi.c.mariadb~ php-7.4.11/ext/mysqli/mysqli_nonapi.c
--- php-7.4.11/ext/mysqli/mysqli_nonapi.c.mariadb~	2020-10-06 12:49:49.534565626 +0200
+++ php-7.4.11/ext/mysqli/mysqli_nonapi.c	2020-10-06 12:53:07.251854500 +0200
@@ -328,7 +328,7 @@ void mysqli_common_connect(INTERNAL_FUNC
 	php_mysqli_set_error(mysql_errno(mysql->mysql), (char *) mysql_error(mysql->mysql));
 
 #if !defined(MYSQLI_USE_MYSQLND)
-	mysql->mysql->reconnect = MyG(reconnect);
+//	mysql->mysql->reconnect = MyG(reconnect);
 #endif
 	unsigned int allow_local_infile = MyG(allow_local_infile);
 	mysql_options(mysql->mysql, MYSQL_OPT_LOCAL_INFILE, (char *)&allow_local_infile);
diff -up php-7.4.11/ext/mysqli/mysqli_prop.c.mariadb~ php-7.4.11/ext/mysqli/mysqli_prop.c
--- php-7.4.11/ext/mysqli/mysqli_prop.c.mariadb~	2020-09-29 12:17:09.000000000 +0200
+++ php-7.4.11/ext/mysqli/mysqli_prop.c	2020-10-06 12:49:49.534565626 +0200
@@ -23,6 +23,7 @@
 
 #include <signal.h>
 
+#include <server/mysql_version.h>
 #include "php.h"
 #include "php_ini.h"
 #include "ext/standard/info.h"
diff -up php-7.4.11/ext/mysqli/php_mysqli_structs.h.mariadb~ php-7.4.11/ext/mysqli/php_mysqli_structs.h
--- php-7.4.11/ext/mysqli/php_mysqli_structs.h.mariadb~	2020-09-29 12:17:09.000000000 +0200
+++ php-7.4.11/ext/mysqli/php_mysqli_structs.h	2020-10-06 12:49:49.534565626 +0200
@@ -73,11 +73,7 @@
 #include <my_sys.h>
 #include <mysql.h>
 #include <errmsg.h>
-#include <my_list.h>
-#include <m_string.h>
 #include <mysqld_error.h>
-#include <my_list.h>
-#include <m_ctype.h>
 #include "mysqli_libmysql.h"
 #endif /* MYSQLI_USE_MYSQLND */
 
