diff -up php-7.4.11/ext/date/php_date.c.0014~ php-7.4.11/ext/date/php_date.c
--- php-7.4.11/ext/date/php_date.c.0014~	2020-10-06 12:53:28.911922273 +0200
+++ php-7.4.11/ext/date/php_date.c	2020-10-06 12:54:00.485354281 +0200
@@ -683,7 +683,7 @@ static zval *date_period_get_property_pt
 static int date_object_compare_timezone(zval *tz1, zval *tz2);
 
 /* {{{ Module struct */
-zend_module_entry date_module_entry = {
+__attribute__((visibility("default"))) zend_module_entry date_module_entry = {
 	STANDARD_MODULE_HEADER_EX,
 	NULL,
 	NULL,
diff -up php-7.4.11/ext/libxml/libxml.c.0014~ php-7.4.11/ext/libxml/libxml.c
--- php-7.4.11/ext/libxml/libxml.c.0014~	2020-09-29 12:17:08.000000000 +0200
+++ php-7.4.11/ext/libxml/libxml.c	2020-10-06 12:53:28.911922273 +0200
@@ -130,7 +130,7 @@ static const zend_function_entry libxml_
 	PHP_FE_END
 };
 
-zend_module_entry libxml_module_entry = {
+__attribute__((visibility("default"))) zend_module_entry libxml_module_entry = {
 	STANDARD_MODULE_HEADER,
 	"libxml",                /* extension name */
 	libxml_functions,        /* extension function list */
diff -up php-7.4.11/ext/pcre/php_pcre.c.0014~ php-7.4.11/ext/pcre/php_pcre.c
--- php-7.4.11/ext/pcre/php_pcre.c.0014~	2020-09-29 12:17:14.000000000 +0200
+++ php-7.4.11/ext/pcre/php_pcre.c	2020-10-06 12:53:28.911922273 +0200
@@ -3058,7 +3058,7 @@ static const zend_function_entry pcre_fu
 	PHP_FE_END
 };
 
-zend_module_entry pcre_module_entry = {
+__attribute__((visibility("default"))) zend_module_entry pcre_module_entry = {
 	STANDARD_MODULE_HEADER,
    "pcre",
 	pcre_functions,
diff -up php-7.4.11/ext/reflection/php_reflection.c.0014~ php-7.4.11/ext/reflection/php_reflection.c
--- php-7.4.11/ext/reflection/php_reflection.c.0014~	2020-09-29 12:17:09.000000000 +0200
+++ php-7.4.11/ext/reflection/php_reflection.c	2020-10-06 12:53:28.915255617 +0200
@@ -6950,7 +6950,7 @@ PHP_MINFO_FUNCTION(reflection) /* {{{ */
 	php_info_print_table_end();
 } /* }}} */
 
-zend_module_entry reflection_module_entry = { /* {{{ */
+__attribute__((visibility("default"))) zend_module_entry reflection_module_entry = { /* {{{ */
 	STANDARD_MODULE_HEADER,
 	"Reflection",
 	reflection_ext_functions,
diff -up php-7.4.11/ext/simplexml/simplexml.c.0014~ php-7.4.11/ext/simplexml/simplexml.c
--- php-7.4.11/ext/simplexml/simplexml.c.0014~	2020-09-29 12:17:07.000000000 +0200
+++ php-7.4.11/ext/simplexml/simplexml.c	2020-10-06 12:53:28.915255617 +0200
@@ -2654,7 +2654,7 @@ static const zend_module_dep simplexml_d
 };
 /* }}} */
 
-zend_module_entry simplexml_module_entry = { /* {{{ */
+__attribute__((visibility("default"))) zend_module_entry simplexml_module_entry = { /* {{{ */
 	STANDARD_MODULE_HEADER_EX, NULL,
 	simplexml_deps,
 	"SimpleXML",
diff -up php-7.4.11/ext/spl/php_spl.c.0014~ php-7.4.11/ext/spl/php_spl.c
--- php-7.4.11/ext/spl/php_spl.c.0014~	2020-09-29 12:17:11.000000000 +0200
+++ php-7.4.11/ext/spl/php_spl.c	2020-10-06 12:53:28.915255617 +0200
@@ -1042,7 +1042,7 @@ PHP_RSHUTDOWN_FUNCTION(spl) /* {{{ */
 
 /* {{{ spl_module_entry
  */
-zend_module_entry spl_module_entry = {
+__attribute__((visibility("default"))) zend_module_entry spl_module_entry = {
 	STANDARD_MODULE_HEADER,
 	"SPL",
 	spl_functions,
diff -up php-7.4.11/ext/standard/basic_functions.c.0014~ php-7.4.11/ext/standard/basic_functions.c
--- php-7.4.11/ext/standard/basic_functions.c.0014~	2020-09-29 12:17:11.000000000 +0200
+++ php-7.4.11/ext/standard/basic_functions.c	2020-10-06 12:53:28.915255617 +0200
@@ -3448,7 +3448,7 @@ static const zend_module_dep standard_de
 };
 /* }}} */
 
-zend_module_entry basic_functions_module = { /* {{{ */
+__attribute__((visibility("default"))) zend_module_entry basic_functions_module = { /* {{{ */
 	STANDARD_MODULE_HEADER_EX,
 	NULL,
 	standard_deps,
diff -up php-7.4.11/Zend/zend_signal.c.0014~ php-7.4.11/Zend/zend_signal.c
--- php-7.4.11/Zend/zend_signal.c.0014~	2020-09-29 12:17:14.000000000 +0200
+++ php-7.4.11/Zend/zend_signal.c	2020-10-06 12:53:28.915255617 +0200
@@ -386,7 +386,7 @@ static void zend_signal_globals_ctor(zen
 }
 /* }}} */
 
-void zend_signal_init(void) /* {{{ */
+__attribute__((visibility("default"))) void zend_signal_init(void) /* {{{ */
 {
 	int signo;
 	struct sigaction sa;
@@ -408,7 +408,7 @@ void zend_signal_init(void) /* {{{ */
 
 /* {{{ zend_signal_startup
  * alloc zend signal globals */
-ZEND_API void zend_signal_startup(void)
+__attribute__((visibility("default"))) ZEND_API void zend_signal_startup(void)
 {
 
 #ifdef ZTS
