diff -up php-7.3.0RC1/ext/dba/config.m4.dba-link.droplet php-7.3.0RC1/ext/dba/config.m4
--- php-7.3.0RC1/ext/dba/config.m4.dba-link.droplet	2018-09-13 22:49:02.287930835 +0200
+++ php-7.3.0RC1/ext/dba/config.m4	2018-09-13 23:19:59.940775252 +0200
@@ -282,6 +282,8 @@ AC_DEFUN([PHP_DBA_DB_CHECK],[
 #endif
           ],[
             THIS_LIBS=$LIB
+            LDFLAGS=$old_LDFLAGS
+            LIBS=$old_LIBS
             lib_found=1
           ])
         ],[])
